src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -2862,6 +2862,75 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/DRSError'
+    post:
+      tags:
+        - DataRepositoryService
+      summary:  Get info about a DrsObject through POST'ing a Passport.
+      description:  Returns object metadata, and a list of access methods that can be used to fetch object bytes.  Method is a POST to accomodate a JWT GA4GH Passport sent in the formData in order to authorize access.
+      operationId: PostObject
+      parameters:
+        - name: object_id
+          in: path
+          required: true
+          schema:
+            type: string
+      security:
+        - PassportAuth: [ ]
```

Comment:
What is this `security` field used for? I haven't seen that before!

Reply:
Thanks for asking about this! I had copied it from the DRS Spec, but overlooked revisiting what it meant. 
- In general, here's more details on the security field in swagger - https://swagger.io/docs/specification/authentication/
- I'm adding a change to add a security schema to define "passportauth"

Reply:
As I mentioned in standup, I decided to remove this security option. As planned, the user will just provide the passport as part of the request and we will make this endpoint unauthenticated.